packages/cli/package.json

```diff
@@ -39,13 +39,15 @@
     "@configu/common": "workspace:*",
     "@configu/sdk": "workspace:*",
     "axios": "^1.7.4",
+    "change-case": "4.1.2",
     "clipanion": "^4.0.0-rc.4",
     "consola": "^3.2.3",
     "inquirer": "12.0.0",
     "lodash": "^4.17.21",
     "open": "10.1.0",
     "openid-client": "5.7.0",
-    "std-env": "^3.7.0"
+    "std-env": "^3.7.0",
+    "tiged": "2.12.7"
```

Comment:
Tiged is a fork of degit, which is still actively maintained (last update [2 months ago](https://github.com/tiged/tiged) rather than [3 years ago](https://github.com/Rich-Harris/degit))

Reply:
👍